fe/fe-core/src/main/java/com/starrocks/common/ErrorCode.java

```diff
@@ -112,7 +112,9 @@ public enum ErrorCode {
             "Client does not support authentication protocol requested by server; consider upgrading MySQL client"),
 
     ERR_ACCESS_DENIED(1252, new byte[] {'4', '2', '0', '0', '0'},
-            "Access denied; you need (at least one of) the %s privilege(s) on %s%s for this operation"),
+            "Access denied; you need (at least one of) the %s privilege(s) on %s%s for this operation. " +
+                    "Please ask the admin to grant permission(s) or try activating existing roles using <set [default] role>. " +
+                    "Current role(s): %s. Inactivated role(s): %s."),
```

Comment:
For ranger access controller, there's no concept of activated or inactivated roles, we should slightly change the error message for different access controller to avoid misunderstanding? 
also @wangsimo0